tensorflow/lite/micro/tools/BUILD

```diff
@@ -88,3 +99,76 @@ py_binary(
         "@absl_py//absl:app",
     ],
 )
+
+py_library(
+    name = "model_transforms_utils",
+    srcs = ["model_transforms_utils.py"],
+    data = [
+        ":tflite_flatbuffer_align_wrapper.so",
+    ],
+    srcs_version = "PY3",
+    visibility = [
+        ":tflm_tools",
+        # TODO(b/285021986): properly fix visibility
```

Comment:
the bug linked from here doesn't give me additional context on what is needed for the visibility fix. If you can add a comment there about exactly what the issue with the current visibility is, and link to that specific comment from here, that would be great.

```py
#TODO(b/285021986): see http://b/285021986#comment2 for details on what is needed to fix the visibility.
```

Reply:
Done.